docs/changelog.rst

```diff
@@ -17,6 +17,8 @@ Changelog
 * :feature:`5588` Users will now be able to save and restore used filters in the history section.
 * :feature:`5587` Forward/Backward navigation for filters/pagination (using mouse buttons) will now be possible in the history section.
 * :feature:`5569` It is now possible to detect when an evm address is used in other chains and start tracking it. Also a background task runs periodically that performs this check.
+* :feature:`3004` Users will now be able to automatically track balances in convex gauges and their CVX locked and staked.
+* :feature:`5611` Users will now see an improved notification if their API keys for Etherscan are missing and will be easier to take action on it.
```

Comment:
```suggestion
* :feature:`5611` Users will now see an improved notification if their API keys for Etherscan are missing and it will be easier to take action on it.
```